new-player/src/main/java/net/newpipe/newplayer/logic/MediaSourceBuilder.kt

```diff
@@ -47,7 +48,7 @@ internal class MediaSourceBuilder
     (
     private val repository: MediaRepository,
     private val mutableErrorFlow: MutableSharedFlow<Exception>,
-    private val httpDataSourceFactory: HttpDataSource.Factory,
+    private val httpDataSourceFactory: DataSource.Factory,
```

Comment:
This should be renamed to `dataSourceFactory`, as the attribute type has been changed.